walkthroughs/mobile-walkthrough/walkthrough.adoc

```diff
@@ -1,8 +1,25 @@
 :sync-example-app-link: https://github.com/aerogear/apollo-voyager-ionic-example
+:idm-name: Keycloak
 
-= Mobile Walkthrough
+= Developing a Data Sync App
 
-In this walkthrough, we will deploy some mobile services, and run an app that can talk to these services.
+This walkthrough shows you how to create a data sync server-side app and protect it using {idm-name}.
+
+Data sync server-side app::
+A data sync app makes sure that data that is shared is consistent over time.
+
+Data sync client app::
+This can be a mobile or web browser which allows users to view and change data in online and offline scenarios.
+
+Mobile Developer Console::
+The Mobile Developer Console allows you bind your mobile, or in this case, web app to a mobile service or data sync app.
+After binding your app, you can download a configuration file to you development environment and write code that uses the mobile service or data sync server side app.
+
+Data Sync consists of a framework that allows you create server-side and client apps that support the following features:
+
+. Offline data changes
+. Conflict resolution
+. Easy integration with other mobile services
```

Comment:
this intro makes me feel like this walkthrough is all about sync, but I don't think it is the case. Perhaps this intro is good to be added to the section where we deploy the sync app?

Reply:
I originally intended to split this into 2 walkthroughs (Intro to MDC, and using sync), but ran out of time. At the same time I wanted to share with Dara, so, it probably isn't mergable, will revisit